src/recaptcha.js

```diff
@@ -10,7 +10,7 @@ const ReCAPTCHA = React.createClass({
     type: PropTypes.oneOf(["image", "audio"]),
     tabindex: PropTypes.number,
     onExpired: PropTypes.func,
-    size: PropTypes.oneOf(["compact", "normal"]),
+    size: PropTypes.oneOf(["compact", "normal", "invisible"]),
```

Comment:
I feel that we should be a new prop called `type` and then release this as a major change.

```type: PropTypes.oneOf(["invisible", "visible"]),```

or we could have a bool `isInvisible` Thoughts?

This is what google calls it type:

<img width="789" alt="recaptcha__easy_on_humans__hard_on_bots" src="https://cloud.githubusercontent.com/assets/462491/23974372/2f77c6ee-0998-11e7-8f54-6ecea6c6b890.png">